### PR TITLE
Adding INVENTORY partition to the protective MBR

### PR DESCRIPTION
**Pull request by rvs**
After a lot of tweaking and trial-and-error with windows and various BIOSes here's a tweak that appears to be minimal and functional. What it is trying to address is the fact that on Windows we need to make sure INVENTORY partition is recognized by the operating system via protective MBR (otherwise it won't be visible when a USB stick is inserted into a computer runnings Windows).

With this in place, protective MBR will now have 4 elements:
   1. EFI boot partition
   2. config partition
   3. INVENTORY partition
   4. Protective MBR entry
 
Before we only had №1 and №2+4 mushed together (don't ask why it made sense -- it did at the time ;-)).

Long story short, with this now in place, both Mac OS X and Windows (including older versions) appear to be happy with reading INVENTORY partition on our USB stick.

This is also the basis for my next proposal -- which is combining №1 and №2 since it really makes much more sense to have the content of the №2 in a folder inside of №1 (hence a bit of a hacky code in this patch -- but the good news is that it is going away real soon).

Finally, given this new scheme -- it became possible to use fdisk instead of the `dd` hackery to adjust protective MBR entries.

### CODE REVIEW

File: pkg/mkimage-raw-efi/make-raw [hunk 1]
```diff
@@ -214,6 +216,8 @@ do_inventory_win() {
     local NUM_PART=$(( PART_OFFSET + 5 ))
     local PART_TYPE=EBD0A0A2-B9E5-4433-87C0-68B6B72699C7
 
+    PROTECTIVE_MBR_LIST="$PROTECTIVE_MBR_LIST$(( PART_OFFSET + 4 )):$NUM_PART:"
```
Comment (giggsoff) [inline pkg/mkimage-raw-efi/make-raw:219]: I can see here information about config partition (PART_OFFSET + 4). Do we need it to be defined inside inventory (and also _win) function?

**Review by eriknordmark — APPROVED** [review]
Have you verified it can be read on Linux as well? (Since Linux is writing it I would assume so, but if you have an image I can verify it on Ubuntu.)
Kicking off the build/tests in any case.

**Comment by rvs** [comment]
> Have you verified it can be read on Linux as well? (Since Linux is writing it I would assume so, but if you have an image I can verify it on Ubuntu.) Kicking off the build/tests in any case.

Yup! Testes it pretty much everywhere @eriknordmark  (including some legacy BIOS boxes -- which is why it took so long b/c they are in my home lab).

File: pkg/mkimage-raw-efi/make-raw (hunk 1 above)
Comment (rvs) [reply pkg/mkimage-raw-efi/make-raw:219]: Like I mentioned in my PR description -- I'd rather not. Yes it is a hack, but it is hack that's going to go away ASAP since I'm working on merging conf partition into EFI one under `/conf` folder. Hence hoping this one line of a hack would not be too much of a problem to merge this PR @giggsoff

Comment (giggsoff) [reply pkg/mkimage-raw-efi/make-raw:219]: Ah, sorry, miss this part of description. Thank you for pointing me onto it.
